src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="card margin-bottom-15px" *ngFor="let question of questionsOrder">
   <div class="card-header bg-info text-white course-tab-header" (click)="toggleAndLoadTab.emit(question.question.feedbackQuestionId)">
-    <tm-question-text-with-info [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails" (click)="$event.stopPropagation()"></tm-question-text-with-info>
+    <tm-question-text-with-info class="tm-question-text-with-info" [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails"
+                                 (click)="expandBoxByTextClick($event)" ></tm-question-text-with-info>
```

Comment:
On click handler here has overlapped functionality with the on click handler at line 2